content/tagging/_index.md

```diff
@@ -17,19 +17,19 @@ further_reading:
 
 ## Introduction
 
-Tags are a way of adding dimensions to metrics, so they can be sliced, diced, aggregated, and compared on the front end. [Using tags][1] enables you to observe aggregate performance across a number of hosts and (optionally) narrow the set further based on specific elements. In a nutshell, tagging is a method to scope aggregated data.
+Tags are a way of adding dimensions to metrics, so they can be sliced, diced, aggregated, and compared on the front end. [Using tags][1] enables you to observe aggregate performance across a number of hosts and (optionally) narrow the set further based on specific elements. In summary, tagging is a method to scope aggregated data.
```

Comment:
> observe aggregate performance

Should this be "observe aggregate data points" instead? Just to interpret it as not only for performance-related metrics.